peewee_extra_fields/__init__.py

```diff
@@ -1112,12 +1112,15 @@ def db_value(self, value):
         return value
 
     def python_value(self, value):
-        return json.loads(value)
+        if not value is None:
+            value = json.loads(value)
```

Comment:
This is done because when the value is NULL from the database, JSON throws an exception